values/qkm.yaml.gotmpl

```diff
@@ -4,9 +4,9 @@ image:
 environment:
   HTTP_HOST: 0.0.0.0
   LOG_FORMAT: json
-  DB_HOST: postgres-postgresql.{{ .Values.qkm.namespace }}
+  DB_HOST: {{ .Values.postgresql.host }}.{{ .Values.qkm.namespace }}
```

Comment:
I believe `namespace` is redundant here, what would it happen in case clients are using an external Postgres?

Reply:
Namespace removed